08LessonEight/homework/exercise-08.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title>Excercise 08</title>
+    <link type="text/css" rel="stylesheet" href="css/exercise-08-style.css" />
+</head>
+
+<body>
+    <div class="bigtruck">
+        <img src="img/truck.jpg" />
+        <h3>Safe & Secured Delivery</h3>
```

Comment:
use `&amp;` for the `&` symbol - that way you won't run into problems when your code is being parsed